File: deploy.py
#!/usr/bin/env python
" Deployer "

import boto3

def deploy():
    " Deploy direct to Lambda and publish "
    client = boto3.client('lambda')
    response = client.update_function_code(
        FunctionName='reposit',
        ZipFile=open('reposit.zip', 'rb').read(),
        Publish=True
    )
    print(response)

if __name__ == '__main__':
    deploy()

File: deploy.js

var fs = require('fs');
var AWS = require('aws-sdk');
var lambda = new AWS.Lambda();

var params = {
    FunctionName: "reposit", 
    Publish: true, 
    ZipFile: fs.readFileSync("reposit.zip")
};
lambda.updateFunctionCode(params, function(err, data) {
    if (err) console.log(err, err.stack); // an error occurred
    else     console.log(data);           // successful response
});